README.md

```diff
@@ -3,23 +3,28 @@
 [![ci](https://github.com/nearform/optic-expo/actions/workflows/ci.yml/badge.svg)](https://github.com/nearform/optic-expo/actions/workflows/ci.yml)
 [![cd](https://github.com/nearform/optic-expo/actions/workflows/cd.yml/badge.svg)](https://github.com/nearform/optic-expo/actions/workflows/cd.yml)
 
-Optic is an app that helps you securely generate OTP tokens for 2FA protected npm accounts. It allows auto-publish npm packages using CI.
+[Optic](https://expo.dev/@nearform/optic-expo) is an app that helps you securely generate OTP tokens for 2FA protected npm accounts. It allows auto-publish npm packages using CI.
 
 ## Requirements
 
 - Node LTS
 - yarn
+- expo go app (on your ios/android phone or you can use the ios simulator)
 
 ## Setup
 
 1. `cp .env.sample .env`
-2. `yarn`
-3. `yarn start`
+1. `yarn`
+1. `yarn start`
 
 ## Development
 In order to successfully run the Optic-expo app locally you will need the following:
 1. Expo user account. You can sign up [here](https://expo.dev/signup).
-2. Once you have an Expo account, your account needs to be added to the NearForm organization (ask @simoneb to do that for you).
+1. Once you have an Expo account, your account needs to be added to the NearForm organization (ask @simoneb to do that for you).
+1. Scan the QR code on your terminal or go to `exp://172.22.22.56:19000`
+1. Once the app loads, if you get the signin screen, in your terminal run `expo login -u <username> -p <password>` and reload the app
```

Comment:
There is [an issue](https://stackoverflow.com/questions/66857797/exnetwork-error-403-when-publishing-app-to-ios-with-expo) with expo in ios, where a user needs to be a member of the expo organisation in order to view the app. There's a good chance that the above doc instruction to do an `expo login`, won't be needed if we fix the issue.